iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -32,399 +32,229 @@
 #include "ts/ink_defs.h"
 #include "ts/hugepages.h"
 
+#include <algorithm>
+
 /// Global singleton.
 class EventProcessor eventProcessor;
 
-class ThreadAffinityInitializer : public Continuation
```

Comment:
he did.  but he was frustrated at how huge it was, as I read his comments.  It does nearly nothing.    All it does is go through a sequence of added callbacks.

No data.  No benefit.  15 times bigger ... and really unrelated to anything that EventProcessor does.  

He says he did it to help you out.